File: mario.py
import cs50
# Keep asking user for inpt until it meets the condition
while True:
    print("Enter the Height of the Pyramid ", end="")
    n = cs50.get_int()
    if n > 0 and n < 23:
        break

# Loop for printing the entire pyramid
for i in range(n):
    # Loop for printing spaces - LEFT side
    for j in range(n - (i+1)):
        print(" ", end="")
    # Loop for printing hash-tag - LEFT side
    for k in range(i + 1):
        print("#", end="")
    # Print middle space - fixed
    print(" ", end="")
    # Loop for printing hash-tag - RIGHT side
    for l in range(i + 1):
        print("#", end="")
    #Loop for printing spaces - RIGHT side
    for m in range(n - (i +1)):
        print(" ", end="")
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

int n;

int main (void)
{
   do
   {
       n = get_int("Enter the Height of the Pyramid ");
   }
   while (n < 0 || n > 23);


  // Loop for printing the entire pyramid
   for (int i = 0; i < n; i++ )
   {
     // Loop for printing spaces - LEFT side
      for (int j = 0; j < (n-(i+1)); j++)
      {
         printf(" ");
      }
      //Loop for printing hash-tag - LEFT side
      for (int k = 0; k <= i; k++)
      {
         printf("#");
      }
      //Print middle space - fixed
      printf(" ");
      //Loop for printing hash-tag - RIGHT side
      for (int l = 0; l <= i; l++)
      {
         printf("#");
      }
      //Loop for printing spaces - RIGHT side
      for (int m = 0; m < (n-(i+1)); m++)
      {
         printf(" ");
      }
     printf("\n");
   }
}
